OneBlink.SDK/FormsAppsClient.cs

```diff
@@ -22,17 +23,14 @@ public FormsAppsClient(string accessKey, string secretKey, TenantName tenantName
 
         public async Task<JWTPayload> VerifyJWT(string token)
         {
+            HttpRequestMessage httpRequestMessage = new HttpRequestMessage(HttpMethod.Get, this.oneBlinkApiClient.tenant.oneBlinkAPIOrigin + "/my-forms-app");
+            await this.oneBlinkApiClient.SendRequest(httpRequestMessage, token);
+
             if (token.Contains("Bearer "))
             {
                 token = token.Split(' ')[1];
             }
-            JwtSecurityToken jwt = new JwtSecurityToken(token);
-            string iss = this.oneBlinkApiClient.tenant.jwtIssuer;
-            JsonWebKey jwk = await Token.GetJsonWebKey(
-                iss: iss,
-                kid: jwt.Header.Kid
-            );
-            string verifiedToken = Token.VerifyJWT(token, jwk, iss);
+            string verifiedToken = Token.DecodeJWT(token);
```

Comment:
```suggestion
           RawJWTPayload rawJWTPayload = JsonConvert.DeserializeObject<RawJWTPayload>(token);
            if (String.IsNullOrEmpty(rawJWTPayload.sub))
            {
                return null;
            }
```

Reply:
`token` is a JWT string, not a JSON string.

Reply:
I was trolling :)